DMTR-311.tex

```diff
@@ -15,17 +24,16 @@
 \title{Characterization Metric Report: Science Pipelines Version 22.0.0}
 % \setDocSubtitle{Optional subtitle}
 
-\author{%
-Jeff Carlin
-}
+\author{Jeff Carlin}
 
 \setDocRef{DMTR-311}
 \setDocUpstreamLocation{\url{https://github.com/lsst-dm/DMTR-311}}
 \date{\vcsDate}
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-Add abstract text.
+This brief report describes measurements of data quality metrics that were carried out for release v22.0.0 of the LSST Science Pipelines.
+The report for the previous version can be found in \citedsp{DMTR-281}.
```

Comment:
On the landing page, this reference does not render

Reply:
Right - I have to update the lsst-tex bibliography to add that...